### PR TITLE
Enable ASG CloudWatch metrics

### PR DESCRIPTION
**Pull request by sean-smith**
Added:
```
"GroupDesiredCapacity"
"GroupInServiceInstances"
"GroupTerminatingInstances"
"GroupTotalInstances"
```

Work done by @stephenmsachs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloudformation/aws-parallelcluster.cfn.json [hunk 1]
```diff
@@ -1642,7 +1642,8 @@
                 "autoscaling:SetDesiredCapacity",
                 "autoscaling:UpdateAutoScalingGroup",
                 "autoscaling:DescribeTags",
-                "autoscaling:SetInstanceHealth"
+                "autoscaling:SetInstanceHealth",
+		"autoscaling:EnableMetricsCollection"
```
Comment (lukeseawalker) [inline cloudformation/aws-parallelcluster.cfn.json]: indent

File: cloudformation/aws-parallelcluster.cfn.json [hunk 2]
```diff
@@ -1642,7 +1642,8 @@
                 "autoscaling:SetDesiredCapacity",
                 "autoscaling:UpdateAutoScalingGroup",
                 "autoscaling:DescribeTags",
-                "autoscaling:SetInstanceHealth"
+                "autoscaling:SetInstanceHealth",
+                "autoscaling:EnableMetricsCollection"
```
Comment (demartinofra) [inline cloudformation/aws-parallelcluster.cfn.json]: This is probably needed in the UserRole not in the RootRole

File: cloudformation/aws-parallelcluster.cfn.json (hunk 1 above)
Comment (sean-smith) [reply cloudformation/aws-parallelcluster.cfn.json]: Removed

**Comment by sean-smith** [comment]
IAM Credentials PR: https://github.com/awsdocs/aws-parallelcluster-user-guide/pull/9

File: cloudformation/aws-parallelcluster.cfn.json (hunk 2 above)
Comment (sean-smith) [reply cloudformation/aws-parallelcluster.cfn.json]: Yep https://github.com/awsdocs/aws-parallelcluster-user-guide/pull/9

**Pull request merged**